frontend/src/components/AccountSequenceContainer.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <ContainerComponent>
+    <template #Header>
+      <h1 class="underline text-xl">Sequenzen des Nutzers:</h1>
+    </template>
+    <div v-for="sequence in sequences" :key="sequence.code">
+      <ContainerComponent>
+        <div class="flex flex-col">
+          <div>
+            {{ sequence.name }}
+          </div>
+          <div class="flex">
+            <ButtonComponent @click="showSequence(sequence.code)">
+              Anzeigen
+            </ButtonComponent>
+            <div class="flex-grow text-red-500">
+              <div v-if="showError">Löschen fehlgeschlagen</div>
+            </div>
+            <ButtonComponent @click="deleteSequence(sequence.code)">
+              Löschen
+            </ButtonComponent>
+          </div>
+        </div>
+      </ContainerComponent>
+    </div>
+  </ContainerComponent>
+</template>
+<script setup lang="ts">
+import { PropType, ref } from 'vue';
+import { ISequence } from '../../../model/sequence/ISequence';
+import ContainerComponent from './ContainerComponent.vue';
+import ButtonComponent from './ButtonComponent.vue';
+import { SequenceRestInterface } from '../restInterfaces/SequenceRestInterface';
+import { router } from '../router';
+
+defineProps({
+  sequences: {
+    type: Object as PropType<Array<ISequence>>,
```

Comment:
Use `Array<ISequence>` as type here

Reply:
i would change it to `ISequence[]` and keep the other, to be consistent with the model

Reply:
The correct way to type arrays in vue is directly setting the type to array, without proptype. So just `type: Array<...>`

Reply:
ah okay then i will go for it